src/Microsoft.ML.TimeSeries/RootCauseAnalyzer.cs

```diff
@@ -603,10 +612,12 @@ private bool StopAnomalyComparison(double preTotal, double parent, double curren
 
         private bool ShouldSeparateAnomaly(double total, double parent, int totalSize, int size)
         {
+            /*
```

Comment:
You want to filter out all the anomaly points? If we remove this part,  lots of insignificant root causes will be return.

Reply:
Sorry, no. This was just left over from a test.